File: jv/datAbs.java
class Persona {
    String nombre;
    int edad;

    public Persona() {

        nombre = "";
        edad = 0;

    }

    public void mostrarDatos(){

        System.out.print("Nombre: "+nombre+", Edad: "+edad);

    }

}

public class datAbs {

    public static void main(String[] args) {

        Persona[] people = new Persona[3];
        people[0] = new Persona();
        people[0].nombre = "David";
        people[0].edad = 19;
        people[0].mostrarDatos(); 

    }
    
}

File: c#/datAbs/datAbs.cs
using System;
using System.ComponentModel;

class Persona
{
    public string nombre;
    public int edad;

    public Persona()
    {
        nombre = "";
        edad = 0;
    }

    public void mostrarDatos()
    {
        Console.WriteLine($"Nombre: {nombre}, Edad:{edad}");
    }

}

class Program
{
    static void Main()
    {
        Persona[] personas = new Persona[3];
        personas[0] = new Persona();
        personas[0].nombre = "David";
        personas[0].edad = 19;
        personas[0].mostrarDatos();
    }
}
